### DISCUSSION TITLE
Is it time to consider renaming Linux Mint to mintOS?

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#483] [Ideas]
I understand that this is a small adjustment, but I believe this name sounds cleaner and less technical. Ultimately, it's just my preference. What do you guys think?

**Comment by claudiux** [comment]
Today is June 4th, not April 1st.

**Comment by SebastJava** [comment]
We have already gone through a long and hard road towards a clean logo...
Personally I would like this change:

**_Linux Mint_** > **_LinuxMint_**

Just remove that " ". That's a common practice: https://en.wikipedia.org/wiki/Camel_case

> Common examples include _YouTube_, _PowerPoint_, _HarperCollins_, _FedEx_, _iPhone_, _eBay_, and _LaGuardia_.

**Comment by Secret-chest** [comment]
Do you want to copy macOS?

**Reply by ghost** [reply]
Would MintOS be better?

**Comment by mtwebster** [comment]
Linux Mint has brand recognition and won't be changing any time soon.
